mysql/assets/configuration/spec.yaml

```diff
@@ -33,7 +33,7 @@ files:
             value:
               type: string
               example: localhost
-              default: None
+              default: ''
```

Comment:
Do we need default here at all? Can it be removed?

Reply:
I think it could be left out- since it defaults to an empty string, it's hard to show that here

Reply:
Updated!